setup.cfg

```diff
@@ -27,4 +27,6 @@ per-file-ignores =
     scripts/*.py:D100,D101,D102,D103,D104,D107,D205,D400,D401,RST201,RST205,RST301
     test/*.py:D104
     test/*/*.py:D100,D101,D102,D103,D104,D105,D107
+    test_othermlpackages/*.py:D104
```

Comment:
not very fond of this name... I presume it's not test/othermlpackages because that would mean you can't run `pytest test/` in the pipelines?

Reply:
Yes, although.... with the `ImportException` guards they should be harmless now.

Reply:
Then... should they just be under `test/` instead?

Reply:
This goes with the other comment.... something has to give. If we keep the `ImportException` guards, then all these can certainly move under `test/`. However, I am more persuaded by your point that those guards could get us spuriously green builds; I'd rather take them out, work around the common code issue, and then keep the separate directory.

Reply:
Indeed! So ignore this comment then :-)